portal-web/test/functional/com/liferay/portalweb/paths/portlet/contentdashboard/ContentDashboard.path

```diff
@@ -194,6 +189,11 @@
 	<td>//*[contains(@class, 'card-body')]//*[contains(@class, 'text-truncate') and contains (text(), '${extensionName}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>FILTER_LABEL</td>
+	<td>//li[contains(@class,'tbar')]//span[contains(@class,'label-item-expand')][starts-with(.,'Category:${filterLabel}')]</td>
```

Comment:
```suggestion
	<td>//li[contains(@class,'tbar')]//span[contains(@class,'label-item-expand')][starts-with(.,'${filterLabel}')]</td>
```

Reply:
There is not only category filter but also other types filter

Reply:
In this case, as I was using `starts-with`, the category text was needed. But I changed to `//li[contains(@class,'tbar')]//span[contains(@class,'label-item-expand') and contains (text(), '${filterLabel}')]` , making it reusable.